src/tests/JIT/Methodical/structs/StructWithSingleFloat.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IlasmRoundTrip>true</IlasmRoundTrip>
```

Comment:
I wonder why this line is necessary. I see a bunch of places that use it, but I'm not sure why.

Reply:
It is not, at least in this project. It is a copy-paste error.